src/components/Sing/SequencerNote.vue

```diff
@@ -279,11 +266,16 @@ const onLeftEdgeMouseDown = (event: MouseEvent) => {
     #{$size} 0 0 $color;
 }
 
+// 共通
 .note {
```

Comment:
（細かいけど）
何が共通かわからないかも。たーーーーぶん何も書かなくても伝わるはず！
```suggestion
.note {
```